lib/src/cmaf/README.md

```diff
@@ -0,0 +1,74 @@
+# CMAF HAM 
+
+## Overview 
+
+HLS and DASH stand as the predominant video streaming technologies currently. Consequently, users often encounter challenges such as converting between HLS and DASH, manipulating manifests, and programmatically understanding manifest structures.
+
+The Common Media Application Format (CMAF) for segmented media (ISO/IEC 23000-19) addresses these challenges by defining a universal format based on ISOBMFF. Additionally, it introduces the Hypothetical Application Model, a framework illustrating the practical usage of CMAF segments and fragments in streaming applications. This project is inspired by the principles outlined in the CMAF standard and the [Hypothetical Application Model] ([Hypothetical Application Model](https://cdn.cta.tech/cta/media/media/resources/standards/cta-5005-a-final.pdf)).
+
+## Features 
+
+* Fundamental on-demand conversion between [HLS] and [DASH] manifests, covering transformations such as HLS to HAM, HAM to HLS, DASH to HAM, and HAM to DASH.
+* Elementary Media Presentation querying functionality.
+* Validation of tracks for enhanced integrity assurance.
+
+## Usage 
+
+Here's a sample demonstration illustrating how to utilize the features of the CMAF HAM module in TypeScript:
+
+
+```typescript
+import {
+  hamToHls,
+  hamToDash,
+  hlsToHam,
+  dashToHam,
+  Presentation,
+  getTracksFromPresentation,
+  validateTracks,
+} from "@svta/common-media-library";
+
+// Validate tracks in a presentation
+const validateTracks = (presentation: Presentation) => {

```

Comment:
Let's keep the validation outside the readme until we improve the validation flow in the PoC